### ISSUE TITLE
DF crashes when launching stonesense

### ISSUE THREAD

**Issue by Scrixx** [DFHack/stonesense#92]
Dwarf Fortress Steam Version

**Specs**
Windows 10
AMD Ryzen 5900x
NVIDIA RTX 3070

**Steps**
1. Install Dwarf Fortress 50.05
2. Download [DFHack 50.05-alpha1](https://github.com/DFHack/dfhack/releases/tag/50.05-alpha1)
3. Extract and copy contents to DF folder and replace SDL.dll file
4. Start a world, embark.
5. In console type stonesense
6. Crash to desktop

The only thing in the error log is this
```
Midmap effective coordinate check out of bounds
Midmap effective coordinate check out of bounds
Midmap effective coordinate check out of bounds
Midmap effective coordinate check out of bounds
```

I tried searching old issues and found a 2018 one that says to set NVDIA control panel to use NVIDIA GPU, I tried turning off extra monitors, I tried fullscreen/windowed, everything still crashes. Please help.

**Comment by lethosor** [comment]
What exactly does the behavior look like? Does the Stonesense window appear at all (e.g. #63)? If so, for how long? Does anything appear in the DFHack console?

**Comment by Scrixx** [comment]
I seem to be getting the same error as that one, something about sRGB profile, but my DF crashes regardless of window size, I've tried fullscreen/windowed with various resolutions. DF just closes after stonesense opens. 

If I'm not in an embark then stonesense opens to this image below
![image](https://user-images.githubusercontent.com/7775947/213397451-8d813cd4-5d19-4b3b-87ec-a3ed1a558eaf.png)

If I'm in an embark, paused or play, then stonesense opens to this below, then everything closes after 2~ seconds.
![image](https://user-images.githubusercontent.com/7775947/213397268-a630e9b2-7ba8-4aba-a5f5-0c144e5aa73f.png)


**stderr**

`DFHack build: 50.05-alpha1-0-g3958d376
Starting with working directory: C:\Users\Scrixx\Downloads\Dwarf.Fortress.v50.05\Dwarf.Fortress.v50.05
Identifying DF version.
Loading hack\symbols.xml ... OK
v0.50.05-steam SDL win64 (windows): PE: 0x63B6063B
v0.50.04-steam SDL win64 (windows): PE: 0x63a2a9ad
Loaded 2 DF symbol tables.
Version: v0.50.05-steam SDL win64
Initializing Console.
Console is running.
Binding to SDL.
Initializing textures.
Initializing plugins.
loading plugin autobutcher
loaded plugin autobutcher; DFHack build 50.05-alpha1-0-g3958d376
loading plugin autochop
loaded plugin autochop; DFHack build 50.05-alpha1-0-g3958d376
loading plugin autofarm
loaded plugin autofarm; DFHack build 50.05-alpha1-0-g3958d376
loading plugin blueprint
loaded plugin blueprint; DFHack build 50.05-alpha1-0-g3958d376
loading plugin changelayer
loaded plugin changelayer; DFHack build 50.05-alpha1-0-g3958d376
loading plugin changevein
loaded plugin changevein; DFHack build 50.05-alpha1-0-g3958d376
loading plugin cleanconst
loaded plugin cleanconst; DFHack build 50.05-alpha1-0-g3958d376
loading plugin cleaners
loaded plugin cleaners; DFHack build 50.05-alpha1-0-g3958d376
loading plugin cleanowned
loaded plugin cleanowned; DFHack build 50.05-alpha1-0-g3958d376
loading plugin cursecheck
loaded plugin cursecheck; DFHack build 50.05-alpha1-0-g3958d376
loading plugin debug
loaded plugin debug; DFHack build 50.05-alpha1-0-g3958d376
loading plugin dig-now
loaded plugin dig-now; DFHack build 50.05-alpha1-0-g3958d376
loading plugin eventful
loaded plugin eventful; DFHack build 50.05-alpha1-0-g3958d376
loading plugin fastdwarf
loaded plugin fastdwarf; DFHack build 50.05-alpha1-0-g3958d376
loading plugin filltraffic
loaded plugin filltraffic; DFHack build 50.05-alpha1-0-g3958d376
loading plugin hotkeys
loaded plugin hotkeys; DFHack build 50.05-alpha1-0-g3958d376
loading plugin liquids
loaded plugin liquids; DFHack build 50.05-alpha1-0-g3958d376
loading plugin misery
loaded plugin misery; DFHack build 50.05-alpha1-0-g3958d376
loading plugin nestboxes
loaded plugin nestboxes; DFHack build 50.05-alpha1-0-g3958d376
loading plugin orders
loaded plugin orders; DFHack build 50.05-alpha1-0-g3958d376
loading plugin overlay
loaded plugin overlay; DFHack build 50.05-alpha1-0-g3958d376
loading plugin pathable
loaded plugin pathable; DFHack build 50.05-alpha1-0-g3958d376
loading plugin probe
loaded plugin probe; DFHack build 50.05-alpha1-0-g3958d376
loading plugin regrass
loaded plugin regrass; DFHack build 50.05-alpha1-0-g3958d376
loading plugin RemoteFortressReader
loaded plugin RemoteFortressReader; DFHack build 50.05-alpha1-0-g3958d376
loading plugin reveal
loaded plugin reveal; DFHack build 50.05-alpha1-0-g3958d376
loading plugin seedwatch
loaded plugin seedwatch; DFHack build 50.05-alpha1-0-g3958d376
loading plugin showmood
loaded plugin showmood; DFHack build 50.05-alpha1-0-g3958d376
loading plugin stonesense
loaded plugin stonesense; DFHack build 50.05-alpha1-0-g3958d376
loading plugin tiletypes
loaded plugin tiletypes; DFHack build 50.05-alpha1-0-g3958d376
loading plugin xlsxreader
loaded plugin xlsxreader; DFHack build 50.05-alpha1-0-g3958d376
Starting the TCP listener.
Starting IO thread.
Starting DF input capture thread.
Listening on port 5000
DFHack is running.
Loading script: dfhack-config/init/default.dfhack.init
Invoking: script hack/init/dfhack.default.init
Loading script: hack/init/dfhack.default.init
Invoking: script hack/init/dfhack.keybindings.init
Loading script: hack/init/dfhack.keybindings.init
Invoking: keybinding add ` gui/launcher
Invoking: keybinding add Ctrl-Shift-D gui/launcher
Invoking: keybinding add Ctrl-Shift-P "gui/launcher --minimal"
Invoking: keybinding add Ctrl-Shift-C hotkeys
Invoking: keybinding add Ctrl-Shift-K gui/cp437-table
Invoking: script hack/init/dfhack.tools.init
Loading script: hack/init/dfhack.tools.init
Invoking: enable overlay
Loading script: dfhack-config/init/dfhack.init
Invoking: gui/prerelease-warning
loaded map in prerelease build
Loading script: dfhack-config/init/default.onLoad.init
Invoking: script hack/init/onLoad.default.init
Loading script: hack/init/onLoad.default.init
Loading script: dfhack-config/init/onLoad.init
Invoking: stonesense
libpng warning: iCCP: known incorrect sRGB profile
libpng warning: iCCP: known incorrect sRGB profile
libpng warning: iCCP: known incorrect sRGB profile
libpng warning: iCCP: known incorrect sRGB profile
libpng warning: iCCP: known incorrect sRGB profile
libpng warning: iCCP: known incorrect sRGB profile
libpng warning: iCCP: known incorrect sRGB profile
libpng warning: iCCP: known incorrect sRGB profile
libpng warning: iCCP: out-of-date sRGB profile with no signature
Loading script: dfhack-config/init/default.onMapLoad.init
Invoking: script hack/init/onMapLoad.default.init
Loading script: hack/init/onMapLoad.default.init
Loading script: dfhack-config/init/onMapLoad.init
Invoking: stonesense
libpng warning: iCCP: known incorrect sRGB profile
libpng warning: iCCP: known incorrect sRGB profile
libpng warning: iCCP: known incorrect sRGB profile
libpng warning: iCCP: known incorrect sRGB profile
libpng warning: iCCP: known incorrect sRGB profile
libpng warning: iCCP: known incorrect sRGB profile
libpng warning: iCCP: known incorrect sRGB profile
libpng warning: iCCP: known incorrect sRGB profile
libpng warning: iCCP: out-of-date sRGB profile with no signature
`

**stdout.log**

`Resetting textures
Resetting textures
Resetting textures
Resetting textures
Resetting textures
Resetting textures
Resetting textures
Resetting textures
Resetting textures
`

**stdout.txt**

`Loading bindings from prefs/interface.txt
Loading bindings from data/init/interface.txt
New window size: 1200x800
Font size: 8x12
Resizing grid to 150x66
Resizing font to 8x12

Resetting textures
New window size: 1200x800
Font size: 8x12
Resizing grid to 150x66
Resizing font to 8x12

dfhack: redirecting stdout to stdout.log (again)
`

**Comment by mkoponen** [comment]
> I seem to be getting the same error as that one, something about sRGB profile

As I mentioned in my issue, although the sRGB warning was the only line of interest either in the logs or the stderr, I determined it to be almost certainly irrelevant to the crash. Because it also occurred in non-crash scenarios. That is to say, if I allowed the game to proceed a bit from the problem save, and then went through the same process. I had the warning and no crash. In other words, my conclusion is that there is nothing informative about the crash in any output.
